plugins/modem-manager/fu-mm-utils.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2019 Aleksander Morgado <aleksander@aleksander.es>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#ifndef __FU_MM_UTILS_H
+#define __FU_MM_UTILS_H
+
+#include "config.h"
+#include <gudev/gudev.h>
+
+G_BEGIN_DECLS
+
+#ifndef HAVE_GUDEV_232
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunused-function"
+G_DEFINE_AUTOPTR_CLEANUP_FUNC(GUdevClient, g_object_unref)
+G_DEFINE_AUTOPTR_CLEANUP_FUNC(GUdevDevice, g_object_unref)
+#pragma clang diagnostic pop
+#endif
+
+gboolean	fu_mm_utils_get_udev_port_info	(GUdevDevice  *dev,
+						 gchar	     **device_sysfs_path,
```

Comment:
tabs before `**device_sysfs_path` pls [1]

[1] I'm not saying it's always sensible, but I think it's important to be consistent :)

Reply:
I'm confused, that line with **device_sysfs_path does have tabs; where are they missing? Is it that all the parameter names should be aligned to the next tab?

Reply:
I've aligned parameter names to the next tab, I hope that was what you asked for :D